fetch.bs

```diff
@@ -813,23 +802,27 @@ the empty string,
 "<code>worker</code>", or
 "<code>xslt</code>". Unless stated otherwise it is the empty string.
 
+<p>A <dfn for=export>script-like destination</dfn> is "<code>script</code>",
+"<code>serviceworker</code>", "<code>sharedworker</code>", or "<code>worker</code>".
+
+<!-- Dependencies:
+     * CSP: https://w3c.github.io/webappsec-csp/#effective-directive-for-a-request
+     * Mixed Content: https://w3c.github.io/webappsec-mixed-content/#should-block-fetch
+     * SRI: https://w3c.github.io/webappsec-subresource-integrity/#apply-algorithm-to-request
+     * HTML -->
```

Comment:
https://w3c.github.io/preload/ also relies on request destination (in the "appropriate times to obtain a resource" section)